include/account.h

```diff
@@ -262,17 +262,17 @@ class CoreExport IdentifyRequest
 	 * for the request to complete. Multiple modules may hold a request at any time,
 	 * but the request is not complete until every module has released it. If you do not
 	 * require holding this (eg, your password check is done in this thread and immediately)
-	 * then you don't need to hold the request before Successing it.
+	 * then you don't need to hold the request before Succeeding it.
```

Comment:
This is probably wrong, but, the original is messy, possibly:
```suggestion
	 * then you don't need to hold the request before `Success`ing it.
```